src/components/Menu/menu.css

```diff
@@ -0,0 +1,29 @@
+.menu-page {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 87vh;
+  padding: 0px;
+}
+
+.menu-page li {
+  margin-bottom: 20px;
+  list-style: none;
+}
+
+.menu-links {
+  padding-bottom: 5px;
+  padding-top: 5px;
+  background-color: black;
+  border-radius: 30px;
+  width: 400px;
+  text-align: center;
+  display: block;
+  text-decoration: none;
+  color: white;
```

Comment:
peace of fucking shit!
1) where mobile sizes?
2) where did you find "color" and bgcolor?
3) top-bottom paddings have to be in one rule.